doc/getting_started/index.rst

```diff
@@ -3,319 +3,372 @@
+you'll:
 
-.. tip::
-
-   Need help with something? See :ref:`help`.
+#. **Set up a command-line development environment** for Linux*, macOS*, or
+   Windows*, with required compiler and build-system tools,
+#. **Get the sources**,
+#. **Build, flash, and run** a sample application on your target board.
 
 .. _host_setup:
 
-Install Host Dependencies
-*************************
+Set up development environment
+******************************
+
+Select and Update OS
+====================
+
+Zephyr development depends on up-to-date releases of common build system
+tools so first, make sure you're development system OS is updated:
+
+.. tabs::
 
-.. _python-pip:
+   .. group-tab:: Linux
 
-Python and pip
-==============
+      Use these commands to bring your Linux (Ubuntu) system up to date.
+      If you're using a different Linux distribution, use the appropriate
+      package manager for your OS.
 
-Python 3 and its package manager, pip\ [#pip]_, are used extensively by Zephyr
-to install and run scripts that are required to compile and run Zephyr
-applications.
+      .. code-block:: bash
 
-Depending on your operating system, you may or may not need to provide the
-``--user`` flag to the ``pip3`` command when installing new packages. This is
-documented throughout the instructions.
-See `Installing Packages`_ in the Python Packaging User Guide for more
-information about pip\ [#pip]_, including this `information on -\\-user`_.
+         sudo apt update
+         sudo apt upgrade
 
-- On Linux, make sure ``~/.local/bin`` is on your :envvar:`PATH`
-  :ref:`environment variable <env_vars>`, or programs installed with ``--user``
-  won't be found\ [#linux_user]_.
+   .. group-tab:: macOS
 
-- On macOS, `Homebrew disables -\\-user`_.
+      On macOS Mojave, you can manually check for updates by choosing
+      System Preferences from the Apple menu, then click Software Update (and
+      click Update Now if there are). For other macOS versions, see the
+      `Update macOS topic in Apple support
+      <https://support.apple.com/en-us/HT201541>`_.
 
-- On Windows, see the Installing Packages information on ``--user`` if you
-  require using this option.
+   .. group-tab:: Windows
 
-On all operating systems, the ``-U`` flag installs or updates the package if the
-package is already installed locally but a more recent version is available. It
-is good practice to use this flag if the latest version of a package is
-required.
+      On Windows, you can manually check for updates by selecting Start  > Settings  >
+      Update & Security  > Windows Update, and then select Check for updates.
+      If updates are available, install them.
 
 .. _install-required-tools:
 
-Install the required tools
-===========================
+Install dependencies
+====================
 
-Follow an operating system specific guide, then come back to this page.
+Next, use your OS package manager to install required OS-provided
```

Comment:
This is tricky because neither homebrew nor chocolatey are OS-provided package managers. They have to be installed on top of the OS as a separate application. Needs specifying what is what.